core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3381,8 +3381,10 @@ private void setAttributeSingleFile(RpcContext rpcContext, LockedInodePath inode
     }
     UpdateInodeEntry.Builder entry = UpdateInodeEntry.newBuilder().setId(inode.getId());
     if (protoOptions.hasReplicationMax() || protoOptions.hasReplicationMin()) {
-      Integer replicationMax = protoOptions.getReplicationMax();
-      Integer replicationMin = protoOptions.getReplicationMin();
+      Integer replicationMax =
+          protoOptions.hasReplicationMax() ? protoOptions.getReplicationMax() : null;
+      Integer replicationMin =
+          protoOptions.hasReplicationMin() ? protoOptions.getReplicationMin() : null;
```

Comment:
Could this logic be written in a more concise way?

Reply:
protobuf defaults these fields to 0 which has meaning for the replication logic. Invoked code expects only explicitly being set values to be non-null, so there isn't much I can do at this layer.